robbobfrh84/bobs-react-challenge-project-jan-2022
# fix(order-form): submit value change

## Changes
1. Changes the scope of the `menuItemChosen` event to include the DOM objects `target` object before specifying it's value

## Purpose
After a user selects a menu item, they are able to submit that order successfully to be stored in the Database with the correct item value. 

## Approach
- First, I tested the bug first as a user to check that I'm able to reproduce the bug.  
- Then proceeded to investigate any error messages within the browser's console, or running command lines environments. 
- Through those error messages I was able to determine the react component and html element that was responsible for the bug. 

## Pre-Testing TODOs
- [] Run local environments 
  - docker backend: `docker compose up`
  - Browser Client: `npm start`
- Visit `localhost:3000` in your browser and login (NOTE: any email & password will be accepted)

## Testing Steps
- Select "Order Form" button in the navbar. 
- Select an Item from the "Lunch menu" dropdown, then the "Order It" Button
- Select "View Order" and see your newly selected Item displayed. 

## Learning
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #1
